apps/condo/domains/resident/tasks/helpers/notifyOnPaydayisCurrnetDateValid.js

```diff
@@ -0,0 +1,13 @@
+const dayjs = require('dayjs')
+
+function notifyOnPaydayIsCurrentDateValid (today) {
+    const targetDayOfSendingThePush = dayjs().set('date', 20).day()
+
+    // https://day.js.org/docs/en/get-set/day
+    if ((targetDayOfSendingThePush === 0 || targetDayOfSendingThePush === 6) && ( today.format('DD') > 16 && today.day() === 5)) {
```

Comment:
Why `today.format('DD') > 16`?
If 20-th day is either Saturday or Sunday, then current Friday can be either 19-th or 18-th day.